cmd/orch-tester/orch_tester.go

```diff
@@ -68,6 +70,134 @@ type broadcasterConfig struct {
 	maxPricePerUnit *int
 }
 
+type broadcasterMetrics struct {
```

Comment:
WDYT of moving these `broadcasterMetrics` to a separate file? Could even be in this same directory not to overcomplicate, but just to keep things separate

Reply:
Good idea. Done in 9e7bd6f5025836870000d1cae8994bb23c467a41